democracy_club/assets/scss/styles.scss

```diff
@@ -198,22 +198,25 @@ article > ol, article > ul {
   @extend .ds-stack-smaller;
 }
 
-.ds-card {
+.ds-cta-card {
+  display: flex;
+  flex-direction: column;
   .ds-card-body {
     display: flex;
     flex-direction: column;
-    align-items: center;
-    text-align: center;
-    margin-bottom: $s1;
-    .ds-cta:last-child {
-      @media screen and (min-width:750px) {
-        position: absolute;
-        bottom: $s1;
-      }
-    }
-  } 
+    flex-grow: 1;
+   & :nth-last-child(2) {
```

Comment:
Still curious why this element needed specific targeting.

Reply:
This is just to give some space between the last element (the CTA button) and the second to last element (whatever it is)